torch/nn/modules/dropout.py

```diff
@@ -181,6 +181,45 @@ def forward(self, input):
 
 
 class FeatureAlphaDropout(_DropoutNd):
+    r"""Randomly masks out entire channels (a channel is a feature map, 
+    e.g. the :math:`j`-th channel of the :math:`i`-th sample in the batch input 
+    is a tensor :math:`\text{input}[i, j]`) of the input tensor). Instead of 
+    setting activations to zero, as in regular Dropout, the activations are set 
+    to the negative saturation value of the SELU activation function.
+
+    Each element will be masked independently for each sample on every forward 
+    call with probability :attr:`p` using samples from a Bernoulli distribution.
+    The elements to be masked are randomized on every forward call, and scaled
+    and shifted to maintain zero mean and unit variance.
+
+    Usually the input comes from :class:`nn.Conv3d` modules.
+
+    As described in the paper `Self-Normalizing Neural Networks`_ ,
```

Comment:
What this paragraph talks about is in the other paper, which you should keep, and add a reference (url) below.

In addition, you should add the reference to Self-Normalizing Neural Networks near the mention of SELU.

Reply:
I see, I think I understand what you are saying - you're saying that two papers should be referenced. The first is "Self-Normalizing Neural Networks," which is to be referenced in the first paragraph re: SELU. The second is "Efficient Object Localization Using Convolutional Networks," which should be referenced here.

Just updated the PR based on this.